src/antsibull_changelog/config.py

```diff
@@ -298,6 +298,17 @@ def get_flatmap(self) -> bool | None:
             )
         return self.flatmap
 
+    def update_galaxy(self, version: str) -> None:
```

Comment:
I don't think this should be a member of `CollectionDetails`. Having it as a free function (and not as part of a class!) in a separate `.py` file (as before) would be better.

Reply:
I can make that change, thank you for the suggestion.